woocommerce/woocommerce-ios
# [REST API] Update Product endpoint mappers

Part of: #8555 


## Description

Apologies for the 500+ change PR. A bunch of the changes are from indentation changes. 🙏 

This PR enables all the mappers from `ProductsRemote` to handle JSON with/without data envelope. 

## Changes

The following mappers have been updated along with mock JSON and unit tests.
- ProductMapper
- ProductListMapper
- ProductSkuMapper
- ProductIDMapper
- EntityIDMapper

Added the missing unit tests for the following
- ProductIDMapper
- EntityIDMapper

## Testing instructions
- Login into the app 
- Navigate to the `Products` tab
- Try to add, edit, update and delete a product
- Everything related to `Products` should work as before

## Screenshots
NA

---
